library/src/main/java/com/datatorrent/lib/io/ConsoleOutputOperator.java

```diff
@@ -53,13 +56,14 @@
     public void process(Object t)
     {
       String s;
+      PrintStream stream = isStderr() ? System.err : System.out;
       if (stringFormat == null) {
         s = t.toString();
       } else {
         s = String.format(stringFormat, t);
       }
```

Comment:
I think line 60-64 should be done only if silent | debug == true o/w these lines of code should not be executed